redhat-developer/developers.redhat.com
# [DEVELOPER 5863] Visual Styles and Display Types

### JIRA Issue Link
https://issues.jboss.org/browse/DEVELOPER-5863

### Verification Process
1. Create new assembly
2. See that "Visual Styles" and/or "Choose Display Type" have the values assigned in Google Sheet: https://docs.google.com/spreadsheets/d/1--r2dCbTIMqeDK3yXREtsvv5-_B2UpHAHIEMSfcd7wA/edit#gid=1570929454
